main.py

```diff
@@ -18,23 +27,51 @@
     help="Take temperature in Celsius and convert to Fahrenheit"
 )
 
+# parser.add_argument(
+#     "temperature", type=float,
+#     help="temperature", action="store", nargs='?'
+# )
+#
+# scale_group = parser.add_mutually_exclusive_group(required=False)
+#
+# scale_group.add_argument(
+#     "-f", "--fahrenheit", action="store_true",
+#     help="use flag -f(--fahrenheit) and provide one value for converting"
+# )
+#
+# scale_group.add_argument(
+#     "-c", "--celsius", action="store_true",
+#     help="use flag -c(--celsius) and provide one value for converting"
+# )
+
 args = parser.parse_args()
+# print(sys.argv)
+# print(args)
+# print(f"{args.fahrenheit} Fahrenheit")
 
+if sys.argv[3:]:
+    print("IT IS PROHIBITED!")
+    quit()
 if args.verbose:
     print(str(args) + " It's verbose argument")
-elif args.fahrenheit:
-    # print(f"{args.fahrenheit} Fahrenheit")
+# if args.fahrenheit and args.celsius:
+#     print("IT IS PROHIBITED!")
+#     quit()
+if args.fahrenheit is not None:
+    # print(f"{args.fahrenheit} Fahrenheit in the logic")
     print(fahrenheit_to_celsius(args.fahrenheit))
-elif args.celsius:
-    # print(f"{args.celsius} Celsius")
+    quit()
+if args.celsius is not None:
     print(celsius_to_fahrenheit(args.celsius))
+    quit()
 else:
     answer = str(input("What temperature you want convert?: ")).lower()
 
-    while answer not in (CELSIUM_INPUT + FAH_INPUT):
+    while answer not in (CELSIUM_INPUT + FAH_INPUT + QUIT_INPUT):
         answer = str(input("What temperature you want convert?: ")).lower()
-        if answer in QUIT_INPUT:
-            quit()
+
+    if answer in QUIT_INPUT:
+        quit()
 
     if answer in CELSIUM_INPUT:  # Convert from Celsius to Fahrenheit
         celsius = input("Provide temperature in Celsius: ")
```

Comment:
@MariaMaximova, it is recommended to add 
```
if __name__ == '__main__':
```
to your code so if I import your main file it will not be excecuted and will not yield any strange errors.